README.md

```diff
@@ -1,192 +1,165 @@
+         maxActive="100"
+         minIdle="10"
+         maxWait="10000"
+         initialSize="10"
+         removeAbandonedTimeout="60"
+         removeAbandoned="true"
+         logAbandoned="true"
+         minEvictableIdleTimeMillis="30000"
+         jmxEnabled="true"
+         jdbcInterceptors="org.apache.tomcat.jdbc.pool.interceptor.ConnectionState;
+           org.apache.tomcat.jdbc.pool.interceptor.StatementFinalizer"
+         username="mat"
+         password="mat"
+         driverClassName="com.mysql.jdbc.Driver"
+         url="jdbc:mysql://localhost:3306/MAT_APP_BLANK"/>
 ```
 
-### Compiling MAT code with GWT
-1.	Right click on the MAT project folder, then go to Properties -> GWT -> General Settings and select Use GWT. Make sure that the SDK is using GWT 2.8.1.
-2.	In Eclipse, select GWT Pulldown (Red Google Icon Button) -> [GWT Compile Project].
-
-In the GWT Compile Wizard:
-1.	Browse to find the MAT project.
-2.	Set the Log level to Debug
-3.	Add the Login, Bonnie and MAT Entry Point Modules.
-
-### Run MAT
-1.	Make sure your database server is running
-2.	In the eclipse IDE Select, [Run] -> [Run Configurations] -> [Web Application] -> [New]
-3.	Set Main Class: “com.google.gwt.dev.DevMode”.
-4.	Select [Apply] and followed by [Run].
-
-### Run MAT on App Server
-Note: If MAT is to be run on an application server, the developer will need to run the build to create a .war file.
-
-1. Run `mvn clean install`
-2. After the build has run, the file is placed into a target folder under MAT as follows: `mat/target/MeasureAuthoringTool.war`
-
-### Log in to MAT
-To login to MAT, open MySQL Workbench and run the following queries:
-1.	 `SELECT * FROM USER where USER_ID='Admin'`
-Look at the LOGIN_ID column, the value there is your UserID. The password default is ‘gargleBlaster_10’.
-2.	Navigate to the MAT log in page GUI and use the UserID and password from the previous step and log in to MAT.
-3.	Once logged in, navigate to the [Mat Account] tab and enter the Admin user details under the [Personal Information] tab and the [Security Questions] tab to setup user’s security questions.
-4.	To change the password to something new, use the [Password] tab.
-
-### Important Security Setup With Git Secrets (https://github.com/awslabs/git-secrets)
-
-1. Use brew to install git secrets
-    - `brew install git-secrets`
-2. Clone this repository (you can skip this if you've already cloned it from previous steps)
-    - Note: You may have to reinitialize these hooks each time you clone a new copy of the repo
-3. Follow the instructions for setting up the pre-commit hooks (from https://github.com/awslabs/git-secrets):
-
+Copy mysql jdbc to tomcat lib directory.
 ```
-cd /path/to/bonnie
-git secrets --install
-git secrets --register-aws
+cp ~/.m2/repository/mysql/mysql-connector-java/5.1.6/mysql-connector-java-5.1.6.jar /apache-tomcat-9.X.X/lib
 ```
+#### Build project with maven
+
+ `mvn clean install` you should see BUILD SUCCESS
```

Comment:
Could you please add a requirement for the maven version? Required maven version is set to  3.6.3